tasks/restore_file.yml

```diff
@@ -15,7 +15,10 @@
     wait_for_job: "{{ cohesity_restore_file.wait_for_job | default('yes') }}"
     wait_minutes: "{{ cohesity_restore_file.wait_minutes | default(10) }}"
     overwrite: "{{ cohesity_restore_file.overwrite | default('yes') }}"
+    vm_name: "{{ cohesity_restore_file.vm_name | default('') }}"
```

Comment:
There should be different tasks/playbooks to restore file for different type or have a type parameter introduced. Else task would be confusing , as to which parameter to choose.

Reply:
Done